chaoscenter/web/src/strings/strings.en.yaml

```diff
@@ -360,6 +360,7 @@ executedBy: Executed by
 executedOn: Executed on
 executionDetails: Execution details goes here
 executionError: Execution Error
+expand: expand
```

Comment:
Could you please clarify if there’s a specific reason this has been included, as it doesn't appear to be in use currently?